src/System.Net.HttpListener/src/System.Net.HttpListener.csproj

```diff
@@ -292,13 +288,33 @@
       <Link>Common\Interop\Windows\sspicli\SSPIWrapper.cs</Link>
     </Compile>
   </ItemGroup>
-  <ItemGroup Condition="'$(TargetsWindows)' != 'true' AND '$(TargetGroup)' != 'net461'">
-    <Compile Include="System\Net\HttpListener.Unix.cs" />
-    <Compile Include="System\Net\HttpListenerContext.Unix.cs" />
-    <Compile Include="System\Net\HttpListenerRequest.Unix.cs" />
-    <Compile Include="System\Net\HttpListenerResponse.Unix.cs" />
-    <Compile Include="System\Net\HttpListenerTimeoutManager.Unix.cs" />
+  <ItemGroup Condition="'$(TargetsUnix)' == 'true' AND '$(TargetGroup)' != 'net461'">
+    <Compile Include="System\Net\Unix\HttpEndPointListener.cs" />
```

Comment:
I believe the plan is to use this managed implementation for WinRT as well.  I'd suggest using a Managed folder rather than Unix.